.travis.yml

```diff
@@ -33,6 +35,13 @@ jobs:
         - make test
       env:
         - n=tests
+    - script:
+        - grip server --rpc-port 18202 --http-port 18201 --config ./test/grids.yml &
+        - sleep 5
+        - make test-conformance
+        - go test ./test
```

Comment:
You should be providing a config file here to enable the GRID backend. Currently you are just running the tests on badger.

Reply:
This unit test was added, and uncovered a bug. The new bug had been fixed.